packages/ui/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,83 @@
+import { createContext } from "@layer-lib/react-context";
+import { HTMLAttributes, useState } from "react";
+
+interface TabsContextType {
+  defaultValue: string;
+  value?: string;
+  onValueChange: (val: string) => void;
+}
+
+const [TabsProvider, useTabsContext] = createContext<TabsContextType>();
+
+interface TabsProps extends React.PropsWithChildren, TabsContextType {}
+
+const Tabs = ({ defaultValue, onValueChange, children }: TabsProps) => {
+  const [value, setValue] = useState(defaultValue);
+  return (
+    <TabsProvider
+      defaultValue={defaultValue}
+      value={value}
+      onValueChange={(value: string) => {
+        onValueChange(value);
+        setValue(value);
+      }}
+    >
+      {children}
+    </TabsProvider>
+  );
+};
+
+interface TabsListProps extends HTMLAttributes<HTMLDivElement> {}
```

Comment:
```suggestion
type TabsListProps extends = HTMLAttributes<HTMLDivElement>
```

확장할 것이 없을 때는 `type`을 사용하는 것이 선호하는데 어떨까요 ?!

Reply:
좋습니다!